### DISCUSSION TITLE
Is it possible to change icons and base css theme?

### DISCUSSION THREAD

**Discussion by ghost** [sni/thruk#1320] [Ideas]
Hi, congratulations for this good software. We are using it over 5 years and the only thing need to monitor our servers.

Maybe it is possible to replace all font icons to a single package? This can lead to a consistent view across all pages.
I have replaced (manually) unicons and fontawesome with tablerfont (because it has lots of different icons) as you can see in the screenshot attached.

For a second suggestion, maybe it's better to use a bootstrap or the like based css file? The one included makes difficult to track classes all across .tt files. I know this is difficult (I have done in my enviroment), but only a few changes a time. This will make applying new themes easier (and bootstrap 5 has dark/light theme based on css and js)

Thanks for reading!

![icons-at-lateral-menu](https://github.com/sni/Thruk/assets/61549534/197f79d5-de3d-42f2-b689-99e5ec5505db)

**Comment by sni** [comment]
We have choosen tailwind as css framework, so in order to switch to bootstrap, you have to rewrite every single template. Same goes for the icons, they cannot be changed easily.
You probably know the themes page already: https://thruk.org/documentation/themes.html#_write-your-own-themes But there is an example for a simple theme and i don't think it is utterly complicated.
Depending on how fundamental the things are you want to adjust.

**Reply by ghost** [reply]
Ok, I have been modifying Light.css adding the classes needed, and removing/changing classes in selected .tt files (copying to the custom templates path).
I will do as always.
Thanks for your answer.
